.github/workflows/create-tags.yaml

```diff
@@ -0,0 +1,50 @@
+name: Manual Tag and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Tag Name (e.g., v1.0.0)"
+        required: true
+      release:
+        description: "Trigger Release? (yes/no)"
+        required: true
+        default: "no"
+
+permissions:
+  contents: write
+  actions: write
```

Comment:
Just to make our lives easier later on, can you add a comment on why we need these permissions?